pkg/handler/repositories.go

```diff
@@ -318,3 +325,70 @@ func (rh *RepositoryHandler) deleteRepository(c echo.Context) error {
 	}
 	return c.NoContent(http.StatusNoContent)
 }
+
+// IntrospectRepository godoc
+// @summary 		introspect a repository
+// @ID				introspect
+// @Tags			repositories
+// @Param  			uuid            path    string  true  "Identifier of the Repository"
+// @Param			reset_count     query   bool    false "Reset the failed introspections count"
+// @Success			204 "Introspection was successfully queued"
+// @Failure      	400 {object} ce.ErrorResponse
+// @Failure      	404 {object} ce.ErrorResponse
+// @Failure      	500 {object} ce.ErrorResponse
+// @Router			/repositories/{uuid}/introspect [post]
```

Comment:
the router here shows /introspect, but the router above actually only accepts /introspect/